installation fail mac osx 10.13.6 fatal error: 'openssl/ssl.h' file not found

Hi,

i'm trying to install netdata using [from source](https://docs.netdata.cloud/packaging/installer/#macos) method , when i run the installer i get  "fatal error: 'openssl/ssl.h' file not found", see below. sound like a directory location issue. any idea how to fix this? 

as a side note, i did manage to install netdata using homebrew method however i only get V16

Thank you

`libnetdata/avl/../socket/security.h:20:12: fatal error: 'openssl/ssl.h' file not found
#  include <openssl/ssl.h>
           ^~~~~~~~~~~~~~~
In file included from libnetdata/adaptive_resortable_list/adaptive_resortable_list.c:3:
In file included from libnetdata/adaptive_resortable_list/../simple_pattern/../libnetdata.h:303:
libnetdata/adaptive_resortable_list/../socket/security.h:20:12: fatal error: 'openssl/ssl.h' file not found
#  include <openssl/ssl.h>
           ^~~~~~~~~~~~~~~
1 error generated.
1 error generated.
In file included from collectors/cups.plugin/cups_plugin.c:9:
In file included from collectors/cups.plugin/../../libnetdata/simple_pattern/../libnetdata.h:303:
collectors/cups.plugin/../../libnetdata/socket/security.h:20:12: fatal error: 'openssl/ssl.h' file not found
#  include <openssl/ssl.h>
           ^~~~~~~~~~~~~~~
make[2]: *** [libnetdata/avl/avl.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[2]: *** [libnetdata/adaptive_resortable_list/adaptive_resortable_list.o] Error 1
In file included from libnetdata/config/appconfig.c:3:
In file included from libnetdata/config/../simple_pattern/../libnetdata.h:303:
libnetdata/config/../socket/security.h:20:12: fatal error: 'openssl/ssl.h' file not found
#  include <openssl/ssl.h>
           ^~~~~~~~~~~~~~~
1 error generated.
make[2]: *** [libnetdata/config/appconfig.o] Error 1
1 error generated.
make[2]: *** [collectors/cups.plugin/cups_plugin.o] Error 1
make[1]: *** [all-recursive] Error 1
make: *** [all] Error 2
 FAILED `

Gotcha. Sorry, I got confused about the process. I use brew frequently for python packages. I installed the netdata one without any errors but nothing happens. I also followed the instructions you provide, and here is the output. Thank you so much for helping me. 

Last login: Sat Sep 21 08:27:23 on ttys000
iMac:~$ git clone https://github.com/netdata/netdata.git --depth=10
Cloning into 'netdata'...
remote: Enumerating objects: 1692, done.
remote: Counting objects: 100% (1692/1692), done.
remote: Compressing objects: 100% (1365/1365), done.
remote: Total 1692 (delta 367), reused 1196 (delta 294), pack-reused 0
Receiving objects: 100% (1692/1692), 3.49 MiB | 1.75 MiB/s, done.
Resolving deltas: 100% (367/367), done.
iMac:~$ cd netdata
iMac:netdata jean-nicolasreyt$ autoreconf -ivf
autoreconf: Entering directory `.'
autoreconf: configure.ac: not using Gettext
autoreconf: running: aclocal --force -I build/m4
autoreconf: configure.ac: tracing
autoreconf: configure.ac: not using Libtool
autoreconf: running: /usr/local/Cellar/autoconf/2.69/bin/autoconf --force
autoreconf: running: /usr/local/Cellar/autoconf/2.69/bin/autoheader --force
autoreconf: running: automake --add-missing --copy --force-missing
configure.ac:38: installing './compile'
configure.ac:37: installing './config.guess'
configure.ac:37: installing './config.sub'
configure.ac:33: installing './install-sh'
configure.ac:33: installing './missing'
Makefile.am: installing './depcomp'
autoreconf: Leaving directory `.'
iMac:netdata $ ./configure --prefix=/usr/local/netdata/usr --sysconfdir=/usr/local/netdata/etc --localstatedir=/usr/local/netdata/var --with-zlib --with-math --with-user=netdata CFLAGS="-O2 -I$(brew --prefix)/opt/openssl/include" LDFLAGS="-L$(brew --prefix)/opt/openssl/lib"
checking whether to enable maintainer-specific portions of Makefiles... no
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... ./install-sh -c -d
checking for gawk... no
checking for mawk... no
checking for nawk... no
checking for awk... awk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking how to create a pax tar archive... gnutar
checking whether make supports nested variables... (cached) yes
checking build system type... x86_64-apple-darwin18.7.0
checking host system type... x86_64-apple-darwin18.7.0
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking whether gcc understands -c and -o together... yes
checking whether make supports the include directive... yes (GNU style)
checking dependency style of gcc... gcc3
checking for g++... g++
checking whether we are using the GNU C++ compiler... yes
checking whether g++ accepts -g... yes
checking dependency style of g++... gcc3
checking for pkg-config... /usr/local/bin/pkg-config
checking pkg-config is at least version 0.9.0... yes
checking how to run the C preprocessor... gcc -E
checking for grep that handles long lines and -e... /usr/bin/grep
checking for egrep... /usr/bin/grep -E
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking minix/config.h usability... no
checking minix/config.h presence... no
checking for minix/config.h... no
checking whether it is safe to define __EXTENSIONS__... yes
checking for __attribute__((returns_nonnull))... no
checking for __attribute__((malloc))... no
checking for __attribute__((noreturn))... no
checking for __attribute__((noinline))... no
checking for __attribute__((format))... no
checking for __attribute__((warn_unused_result))... no
checking for struct timespec... yes
checking for clockid_t... yes
checking for library containing clock_gettime... none required
checking for clock_gettime... yes
checking for sched_setscheduler... no
checking for sched_getscheduler... no
checking for sched_getparam... no
checking for sched_get_priority_min... yes
checking for sched_get_priority_max... yes
checking for getpriority... yes
checking for setpriority... yes
checking for nice... yes
checking for recvmmsg... no
checking for int8_t... yes
checking for int16_t... yes
checking for int32_t... yes
checking for int64_t... yes
checking for uint8_t... yes
checking for uint16_t... yes
checking for uint32_t... yes
checking for uint64_t... yes
checking for inline... inline
checking whether strerror_r is declared... yes
checking for strerror_r... yes
checking whether strerror_r returns char *... no
checking for _Generic... yes
checking for __atomic... yes
checking size of void *... 8
checking whether sys/types.h defines makedev... yes
checking for sys/types.h... (cached) yes
checking for netinet/in.h... yes
checking for arpa/nameser.h... yes
checking for netdb.h... yes
checking for resolv.h... yes
checking for sys/prctl.h... no
checking for sys/vfs.h... no
checking for sys/statfs.h... no
checking for sys/statvfs.h... yes
checking for sys/mount.h... yes
checking for accept4... no
checking operating system... macos with id 3
checking if compiler needs -Werror to reject unknown flags... yes
checking whether pthreads work with -pthread... yes
checking for joinable pthread attribute... PTHREAD_CREATE_JOINABLE
checking if more special flags are required for pthreads... -D_THREAD_SAFE
checking for PTHREAD_PRIO_INHERIT... yes
checking for sin in -lm... yes
checking if libm should be used... yes
checking for uv_fs_scandir_next in -luv... no
checking for LZ4_compress_default in -llz4... no
checking for JudyLIns in -lJudy... no
checking for zlib... yes
checking if zlib should be used... yes
checking for uuid... yes
checking for SHA256_Init in -lcrypto... no
checking for json-c... no
checking for json_object_get_type in -ljson... no
checking if netdata dbengine should be used... no
checking if netdata https should be used... no
checking if json-c should be used... no
checking for memory allocator... system
checking for mallopt... no
checking for mallinfo... no
checking for libcap... no
checking if libcap should be used... no
checking if apps.plugin should be enabled... no
checking for libipmimonitoring... no
checking if freeipmi.plugin should be enabled... no
checking for httpConnect2 in -lcups... yes
checking cups/cups.h usability... yes
checking cups/cups.h presence... yes
checking for cups/cups.h... yes
checking for cups-config... /usr/bin/cups-config
checking if cups.plugin should be enabled... yes
checking linux/netfilter/nfnetlink_conntrack.h usability... no
checking linux/netfilter/nfnetlink_conntrack.h presence... no
checking for linux/netfilter/nfnetlink_conntrack.h... no
checking for libnetfilter_acct... no
checking for libmnl... no
checking if nfacct.plugin should be enabled... no
checking for yajl... no
checking for xenstat_init in -lxenstat... no
checking for xenlight... no
checking if xenstat.plugin should be enabled... no
checking linux/perf_event.h usability... no
checking linux/perf_event.h presence... no
checking for linux/perf_event.h... no
checking if perf.plugin should be enabled... no
checking if slabinfo.plugin should be enabled... no
checking for libcrypto... no
checking for libssl... no
checking for libcurl... yes
checking for curl_easy_init in -lcurl... yes
checking for cJSON_free in -laws-cpp-sdk-core... no
checking for Aws::Kinesis::Model::PutRecordRequest in -laws-cpp-sdk-kinesis... no
checking if kinesis backend should be enabled... no
checking for protobuf >= 3... no
checking for snappy::RawCompress in -lsnappy... no
checking for protoc... no
checking for g++... /usr/bin/g++
checking if prometheus remote write backend should be enabled... no
checking for libmongoc-1.0 >= 1.7... no
checking if mongodb backend should be enabled... no
checking for setns... no
checking if cgroup-network can be enabled... no
checking whether C compiler accepts -flto... yes
checking if -flto builds executables... yes
checking if LTO should be enabled... yes
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: creating Makefile
config.status: creating netdata.spec
config.status: creating backends/graphite/Makefile
config.status: creating backends/json/Makefile
config.status: creating backends/Makefile
config.status: creating backends/opentsdb/Makefile
config.status: creating backends/prometheus/Makefile
config.status: creating backends/prometheus/remote_write/Makefile
config.status: creating backends/aws_kinesis/Makefile
config.status: creating backends/mongodb/Makefile
config.status: creating collectors/Makefile
config.status: creating collectors/apps.plugin/Makefile
config.status: creating collectors/cgroups.plugin/Makefile
config.status: creating collectors/charts.d.plugin/Makefile
config.status: creating collectors/checks.plugin/Makefile
config.status: creating collectors/diskspace.plugin/Makefile
config.status: creating collectors/fping.plugin/Makefile
config.status: creating collectors/ioping.plugin/Makefile
config.status: creating collectors/freebsd.plugin/Makefile
config.status: creating collectors/freeipmi.plugin/Makefile
config.status: creating collectors/cups.plugin/Makefile
config.status: creating collectors/idlejitter.plugin/Makefile
config.status: creating collectors/macos.plugin/Makefile
config.status: creating collectors/nfacct.plugin/Makefile
config.status: creating collectors/node.d.plugin/Makefile
config.status: creating collectors/plugins.d/Makefile
config.status: creating collectors/proc.plugin/Makefile
config.status: creating collectors/python.d.plugin/Makefile
config.status: creating collectors/slabinfo.plugin/Makefile
config.status: creating collectors/statsd.plugin/Makefile
config.status: creating collectors/tc.plugin/Makefile
config.status: creating collectors/xenstat.plugin/Makefile
config.status: creating collectors/perf.plugin/Makefile
config.status: creating daemon/Makefile
config.status: creating database/Makefile
config.status: creating database/engine/Makefile
config.status: creating diagrams/Makefile
config.status: creating health/Makefile
config.status: creating health/notifications/Makefile
config.status: creating libnetdata/Makefile
config.status: creating libnetdata/adaptive_resortable_list/Makefile
config.status: creating libnetdata/avl/Makefile
config.status: creating libnetdata/buffer/Makefile
config.status: creating libnetdata/clocks/Makefile
config.status: creating libnetdata/config/Makefile
config.status: creating libnetdata/dictionary/Makefile
config.status: creating libnetdata/eval/Makefile
config.status: creating libnetdata/locks/Makefile
config.status: creating libnetdata/log/Makefile
config.status: creating libnetdata/popen/Makefile
config.status: creating libnetdata/procfile/Makefile
config.status: creating libnetdata/simple_pattern/Makefile
config.status: creating libnetdata/socket/Makefile
config.status: creating libnetdata/statistical/Makefile
config.status: creating libnetdata/storage_number/Makefile
config.status: creating libnetdata/threads/Makefile
config.status: creating libnetdata/url/Makefile
config.status: creating libnetdata/json/Makefile
config.status: creating libnetdata/health/Makefile
config.status: creating registry/Makefile
config.status: creating streaming/Makefile
config.status: creating system/Makefile
config.status: creating tests/Makefile
config.status: creating web/Makefile
config.status: creating web/api/Makefile
config.status: creating web/api/badges/Makefile
config.status: creating web/api/exporters/Makefile
config.status: creating web/api/exporters/shell/Makefile
config.status: creating web/api/exporters/prometheus/Makefile
config.status: creating web/api/formatters/Makefile
config.status: creating web/api/formatters/csv/Makefile
config.status: creating web/api/formatters/json/Makefile
config.status: creating web/api/formatters/ssv/Makefile
config.status: creating web/api/formatters/value/Makefile
config.status: creating web/api/queries/Makefile
config.status: creating web/api/queries/average/Makefile
config.status: creating web/api/queries/des/Makefile
config.status: creating web/api/queries/incremental_sum/Makefile
config.status: creating web/api/queries/max/Makefile
config.status: creating web/api/queries/median/Makefile
config.status: creating web/api/queries/min/Makefile
config.status: creating web/api/queries/ses/Makefile
config.status: creating web/api/queries/stddev/Makefile
config.status: creating web/api/queries/sum/Makefile
config.status: creating web/api/health/Makefile
config.status: creating web/gui/Makefile
config.status: creating web/server/Makefile
config.status: creating web/server/static/Makefile
config.status: creating config.h
config.status: executing depfiles commands
iMac:netdata jean-nicolasreyt$ CFLAGS="-O2 -I$(brew --prefix)/opt/openssl/include" LDFLAGS="-L$(brew --prefix)/opt/openssl/lib" make
/Library/Developer/CommandLineTools/usr/bin/make  all-recursive
Making all in diagrams
make[2]: Nothing to be done for `all'.
Making all in system
if sed \
		-e 's#[@]localstatedir_POST@#/usr/local/netdata/var#g' \
		-e 's#[@]sbindir_POST@#/usr/local/netdata/usr/sbin#g' \
		-e 's#[@]pluginsdir_POST@#/usr/local/netdata/usr/libexec/netdata/plugins.d#g' \
		-e 's#[@]configdir_POST@#/usr/local/netdata/etc/netdata#g' \
		-e 's#[@]libconfigdir_POST@#/usr/local/netdata/usr/lib/netdata/conf.d#g' \
		-e 's#[@]cachedir_POST@#/usr/local/netdata/var/cache/netdata#g' \
		-e 's#[@]registrydir_POST@#/usr/local/netdata/var/lib/netdata/registry#g' \
		-e 's#[@]varlibdir_POST@#/usr/local/netdata/var/lib/netdata#g' \
		edit-config.in > edit-config.tmp; then \
		mv "edit-config.tmp" "edit-config"; \
	else \
		rm -f "edit-config.tmp"; \
		false; \
	fi
if sed \
		-e 's#[@]localstatedir_POST@#/usr/local/netdata/var#g' \
		-e 's#[@]sbindir_POST@#/usr/local/netdata/usr/sbin#g' \
		-e 's#[@]pluginsdir_POST@#/usr/local/netdata/usr/libexec/netdata/plugins.d#g' \
		-e 's#[@]configdir_POST@#/usr/local/netdata/etc/netdata#g' \
		-e 's#[@]libconfigdir_POST@#/usr/local/netdata/usr/lib/netdata/conf.d#g' \
		-e 's#[@]cachedir_POST@#/usr/local/netdata/var/cache/netdata#g' \
		-e 's#[@]registrydir_POST@#/usr/local/netdata/var/lib/netdata/registry#g' \
		-e 's#[@]varlibdir_POST@#/usr/local/netdata/var/lib/netdata#g' \
		netdata-openrc.in > netdata-openrc.tmp; then \
		mv "netdata-openrc.tmp" "netdata-openrc"; \
	else \
		rm -f "netdata-openrc.tmp"; \
		false; \
	fi
if sed \
		-e 's#[@]localstatedir_POST@#/usr/local/netdata/var#g' \
		-e 's#[@]sbindir_POST@#/usr/local/netdata/usr/sbin#g' \
		-e 's#[@]pluginsdir_POST@#/usr/local/netdata/usr/libexec/netdata/plugins.d#g' \
		-e 's#[@]configdir_POST@#/usr/local/netdata/etc/netdata#g' \
		-e 's#[@]libconfigdir_POST@#/usr/local/netdata/usr/lib/netdata/conf.d#g' \
		-e 's#[@]cachedir_POST@#/usr/local/netdata/var/cache/netdata#g' \
		-e 's#[@]registrydir_POST@#/usr/local/netdata/var/lib/netdata/registry#g' \
		-e 's#[@]varlibdir_POST@#/usr/local/netdata/var/lib/netdata#g' \
		netdata.logrotate.in > netdata.logrotate.tmp; then \
		mv "netdata.logrotate.tmp" "netdata.logrotate"; \
	else \
		rm -f "netdata.logrotate.tmp"; \
		false; \
	fi
if sed \
		-e 's#[@]localstatedir_POST@#/usr/local/netdata/var#g' \
		-e 's#[@]sbindir_POST@#/usr/local/netdata/usr/sbin#g' \
		-e 's#[@]pluginsdir_POST@#/usr/local/netdata/usr/libexec/netdata/plugins.d#g' \
		-e 's#[@]configdir_POST@#/usr/local/netdata/etc/netdata#g' \
		-e 's#[@]libconfigdir_POST@#/usr/local/netdata/usr/lib/netdata/conf.d#g' \
		-e 's#[@]cachedir_POST@#/usr/local/netdata/var/cache/netdata#g' \
		-e 's#[@]registrydir_POST@#/usr/local/netdata/var/lib/netdata/registry#g' \
		-e 's#[@]varlibdir_POST@#/usr/local/netdata/var/lib/netdata#g' \
		netdata.service.in > netdata.service.tmp; then \
		mv "netdata.service.tmp" "netdata.service"; \
	else \
		rm -f "netdata.service.tmp"; \
		false; \
	fi
if sed \
		-e 's#[@]localstatedir_POST@#/usr/local/netdata/var#g' \
		-e 's#[@]sbindir_POST@#/usr/local/netdata/usr/sbin#g' \
		-e 's#[@]pluginsdir_POST@#/usr/local/netdata/usr/libexec/netdata/plugins.d#g' \
		-e 's#[@]configdir_POST@#/usr/local/netdata/etc/netdata#g' \
		-e 's#[@]libconfigdir_POST@#/usr/local/netdata/usr/lib/netdata/conf.d#g' \
		-e 's#[@]cachedir_POST@#/usr/local/netdata/var/cache/netdata#g' \
		-e 's#[@]registrydir_POST@#/usr/local/netdata/var/lib/netdata/registry#g' \
		-e 's#[@]varlibdir_POST@#/usr/local/netdata/var/lib/netdata#g' \
		netdata-init-d.in > netdata-init-d.tmp; then \
		mv "netdata-init-d.tmp" "netdata-init-d"; \
	else \
		rm -f "netdata-init-d.tmp"; \
		false; \
	fi
if sed \
		-e 's#[@]localstatedir_POST@#/usr/local/netdata/var#g' \
		-e 's#[@]sbindir_POST@#/usr/local/netdata/usr/sbin#g' \
		-e 's#[@]pluginsdir_POST@#/usr/local/netdata/usr/libexec/netdata/plugins.d#g' \
		-e 's#[@]configdir_POST@#/usr/local/netdata/etc/netdata#g' \
		-e 's#[@]libconfigdir_POST@#/usr/local/netdata/usr/lib/netdata/conf.d#g' \
		-e 's#[@]cachedir_POST@#/usr/local/netdata/var/cache/netdata#g' \
		-e 's#[@]registrydir_POST@#/usr/local/netdata/var/lib/netdata/registry#g' \
		-e 's#[@]varlibdir_POST@#/usr/local/netdata/var/lib/netdata#g' \
		netdata-lsb.in > netdata-lsb.tmp; then \
		mv "netdata-lsb.tmp" "netdata-lsb"; \
	else \
		rm -f "netdata-lsb.tmp"; \
		false; \
	fi
if sed \
		-e 's#[@]localstatedir_POST@#/usr/local/netdata/var#g' \
		-e 's#[@]sbindir_POST@#/usr/local/netdata/usr/sbin#g' \
		-e 's#[@]pluginsdir_POST@#/usr/local/netdata/usr/libexec/netdata/plugins.d#g' \
		-e 's#[@]configdir_POST@#/usr/local/netdata/etc/netdata#g' \
		-e 's#[@]libconfigdir_POST@#/usr/local/netdata/usr/lib/netdata/conf.d#g' \
		-e 's#[@]cachedir_POST@#/usr/local/netdata/var/cache/netdata#g' \
		-e 's#[@]registrydir_POST@#/usr/local/netdata/var/lib/netdata/registry#g' \
		-e 's#[@]varlibdir_POST@#/usr/local/netdata/var/lib/netdata#g' \
		netdata-freebsd.in > netdata-freebsd.tmp; then \
		mv "netdata-freebsd.tmp" "netdata-freebsd"; \
	else \
		rm -f "netdata-freebsd.tmp"; \
		false; \
	fi
if sed \
		-e 's#[@]localstatedir_POST@#/usr/local/netdata/var#g' \
		-e 's#[@]sbindir_POST@#/usr/local/netdata/usr/sbin#g' \
		-e 's#[@]pluginsdir_POST@#/usr/local/netdata/usr/libexec/netdata/plugins.d#g' \
		-e 's#[@]configdir_POST@#/usr/local/netdata/etc/netdata#g' \
		-e 's#[@]libconfigdir_POST@#/usr/local/netdata/usr/lib/netdata/conf.d#g' \
		-e 's#[@]cachedir_POST@#/usr/local/netdata/var/cache/netdata#g' \
		-e 's#[@]registrydir_POST@#/usr/local/netdata/var/lib/netdata/registry#g' \
		-e 's#[@]varlibdir_POST@#/usr/local/netdata/var/lib/netdata#g' \
		netdata.plist.in > netdata.plist.tmp; then \
		mv "netdata.plist.tmp" "netdata.plist"; \
	else \
		rm -f "netdata.plist.tmp"; \
		false; \
	fi
Making all in tests
if sed \
		-e 's#[@]localstatedir_POST@#/usr/local/netdata/var#g' \
		-e 's#[@]sbindir_POST@#/usr/local/netdata/usr/sbin#g' \
		-e 's#[@]pluginsdir_POST@#/usr/local/netdata/usr/libexec/netdata/plugins.d#g' \
		-e 's#[@]configdir_POST@#/usr/local/netdata/etc/netdata#g' \
		-e 's#[@]libconfigdir_POST@#/usr/local/netdata/usr/lib/netdata/conf.d#g' \
		-e 's#[@]cachedir_POST@#/usr/local/netdata/var/cache/netdata#g' \
		-e 's#[@]registrydir_POST@#/usr/local/netdata/var/lib/netdata/registry#g' \
		-e 's#[@]varlibdir_POST@#/usr/local/netdata/var/lib/netdata#g' \
		health_mgmtapi/health-cmdapi-test.sh.in > health_mgmtapi/health-cmdapi-test.sh.tmp; then \
		mv "health_mgmtapi/health-cmdapi-test.sh.tmp" "health_mgmtapi/health-cmdapi-test.sh"; \
	else \
		rm -f "health_mgmtapi/health-cmdapi-test.sh.tmp"; \
		false; \
	fi
if sed \
		-e 's#[@]localstatedir_POST@#/usr/local/netdata/var#g' \
		-e 's#[@]sbindir_POST@#/usr/local/netdata/usr/sbin#g' \
		-e 's#[@]pluginsdir_POST@#/usr/local/netdata/usr/libexec/netdata/plugins.d#g' \
		-e 's#[@]configdir_POST@#/usr/local/netdata/etc/netdata#g' \
		-e 's#[@]libconfigdir_POST@#/usr/local/netdata/usr/lib/netdata/conf.d#g' \
		-e 's#[@]cachedir_POST@#/usr/local/netdata/var/cache/netdata#g' \
		-e 's#[@]registrydir_POST@#/usr/local/netdata/var/lib/netdata/registry#g' \
		-e 's#[@]varlibdir_POST@#/usr/local/netdata/var/lib/netdata#g' \
		acls/acl.sh.in > acls/acl.sh.tmp; then \
		mv "acls/acl.sh.tmp" "acls/acl.sh"; \
	else \
		rm -f "acls/acl.sh.tmp"; \
		false; \
	fi
if sed \
		-e 's#[@]localstatedir_POST@#/usr/local/netdata/var#g' \
		-e 's#[@]sbindir_POST@#/usr/local/netdata/usr/sbin#g' \
		-e 's#[@]pluginsdir_POST@#/usr/local/netdata/usr/libexec/netdata/plugins.d#g' \
		-e 's#[@]configdir_POST@#/usr/local/netdata/etc/netdata#g' \
		-e 's#[@]libconfigdir_POST@#/usr/local/netdata/usr/lib/netdata/conf.d#g' \
		-e 's#[@]cachedir_POST@#/usr/local/netdata/var/cache/netdata#g' \
		-e 's#[@]registrydir_POST@#/usr/local/netdata/var/lib/netdata/registry#g' \
		-e 's#[@]varlibdir_POST@#/usr/local/netdata/var/lib/netdata#g' \
		urls/request.sh.in > urls/request.sh.tmp; then \
		mv "urls/request.sh.tmp" "urls/request.sh"; \
	else \
		rm -f "urls/request.sh.tmp"; \
		false; \
	fi
if sed \
		-e 's#[@]localstatedir_POST@#/usr/local/netdata/var#g' \
		-e 's#[@]sbindir_POST@#/usr/local/netdata/usr/sbin#g' \
		-e 's#[@]pluginsdir_POST@#/usr/local/netdata/usr/libexec/netdata/plugins.d#g' \
		-e 's#[@]configdir_POST@#/usr/local/netdata/etc/netdata#g' \
		-e 's#[@]libconfigdir_POST@#/usr/local/netdata/usr/lib/netdata/conf.d#g' \
		-e 's#[@]cachedir_POST@#/usr/local/netdata/var/cache/netdata#g' \
		-e 's#[@]registrydir_POST@#/usr/local/netdata/var/lib/netdata/registry#g' \
		-e 's#[@]varlibdir_POST@#/usr/local/netdata/var/lib/netdata#g' \
		alarm_repetition/alarm.sh.in > alarm_repetition/alarm.sh.tmp; then \
		mv "alarm_repetition/alarm.sh.tmp" "alarm_repetition/alarm.sh"; \
	else \
		rm -f "alarm_repetition/alarm.sh.tmp"; \
		false; \
	fi
Making all in backends
Making all in graphite
make[3]: Nothing to be done for `all'.
Making all in json
make[3]: Nothing to be done for `all'.
Making all in opentsdb
make[3]: Nothing to be done for `all'.
Making all in prometheus
Making all in remote_write
make[4]: Nothing to be done for `all'.
make[4]: Nothing to be done for `all-am'.
Making all in aws_kinesis
make[3]: Nothing to be done for `all'.
Making all in mongodb
make[3]: Nothing to be done for `all'.
make[3]: Nothing to be done for `all-am'.
Making all in collectors
Making all in plugins.d
make[4]: Nothing to be done for `all-am'.
Making all in apps.plugin
make[3]: Nothing to be done for `all'.
Making all in cgroups.plugin
if sed \
		-e 's#[@]localstatedir_POST@#/usr/local/netdata/var#g' \
		-e 's#[@]sbindir_POST@#/usr/local/netdata/usr/sbin#g' \
		-e 's#[@]pluginsdir_POST@#/usr/local/netdata/usr/libexec/netdata/plugins.d#g' \
		-e 's#[@]configdir_POST@#/usr/local/netdata/etc/netdata#g' \
		-e 's#[@]libconfigdir_POST@#/usr/local/netdata/usr/lib/netdata/conf.d#g' \
		-e 's#[@]cachedir_POST@#/usr/local/netdata/var/cache/netdata#g' \
		-e 's#[@]registrydir_POST@#/usr/local/netdata/var/lib/netdata/registry#g' \
		-e 's#[@]varlibdir_POST@#/usr/local/netdata/var/lib/netdata#g' \
		cgroup-name.sh.in > cgroup-name.sh.tmp; then \
		mv "cgroup-name.sh.tmp" "cgroup-name.sh"; \
	else \
		rm -f "cgroup-name.sh.tmp"; \
		false; \
	fi
Making all in charts.d.plugin
if sed \
		-e 's#[@]localstatedir_POST@#/usr/local/netdata/var#g' \
		-e 's#[@]sbindir_POST@#/usr/local/netdata/usr/sbin#g' \
		-e 's#[@]pluginsdir_POST@#/usr/local/netdata/usr/libexec/netdata/plugins.d#g' \
		-e 's#[@]configdir_POST@#/usr/local/netdata/etc/netdata#g' \
		-e 's#[@]libconfigdir_POST@#/usr/local/netdata/usr/lib/netdata/conf.d#g' \
		-e 's#[@]cachedir_POST@#/usr/local/netdata/var/cache/netdata#g' \
		-e 's#[@]registrydir_POST@#/usr/local/netdata/var/lib/netdata/registry#g' \
		-e 's#[@]varlibdir_POST@#/usr/local/netdata/var/lib/netdata#g' \
		charts.d.plugin.in > charts.d.plugin.tmp; then \
		mv "charts.d.plugin.tmp" "charts.d.plugin"; \
	else \
		rm -f "charts.d.plugin.tmp"; \
		false; \
	fi
Making all in checks.plugin
make[3]: Nothing to be done for `all'.
Making all in cups.plugin
make[3]: Nothing to be done for `all'.
Making all in diskspace.plugin
make[3]: Nothing to be done for `all'.
Making all in fping.plugin
if sed \
		-e 's#[@]localstatedir_POST@#/usr/local/netdata/var#g' \
		-e 's#[@]sbindir_POST@#/usr/local/netdata/usr/sbin#g' \
		-e 's#[@]pluginsdir_POST@#/usr/local/netdata/usr/libexec/netdata/plugins.d#g' \
		-e 's#[@]configdir_POST@#/usr/local/netdata/etc/netdata#g' \
		-e 's#[@]libconfigdir_POST@#/usr/local/netdata/usr/lib/netdata/conf.d#g' \
		-e 's#[@]cachedir_POST@#/usr/local/netdata/var/cache/netdata#g' \
		-e 's#[@]registrydir_POST@#/usr/local/netdata/var/lib/netdata/registry#g' \
		-e 's#[@]varlibdir_POST@#/usr/local/netdata/var/lib/netdata#g' \
		fping.plugin.in > fping.plugin.tmp; then \
		mv "fping.plugin.tmp" "fping.plugin"; \
	else \
		rm -f "fping.plugin.tmp"; \
		false; \
	fi
Making all in ioping.plugin
if sed \
		-e 's#[@]localstatedir_POST@#/usr/local/netdata/var#g' \
		-e 's#[@]sbindir_POST@#/usr/local/netdata/usr/sbin#g' \
		-e 's#[@]pluginsdir_POST@#/usr/local/netdata/usr/libexec/netdata/plugins.d#g' \
		-e 's#[@]configdir_POST@#/usr/local/netdata/etc/netdata#g' \
		-e 's#[@]libconfigdir_POST@#/usr/local/netdata/usr/lib/netdata/conf.d#g' \
		-e 's#[@]cachedir_POST@#/usr/local/netdata/var/cache/netdata#g' \
		-e 's#[@]registrydir_POST@#/usr/local/netdata/var/lib/netdata/registry#g' \
		-e 's#[@]varlibdir_POST@#/usr/local/netdata/var/lib/netdata#g' \
		ioping.plugin.in > ioping.plugin.tmp; then \
		mv "ioping.plugin.tmp" "ioping.plugin"; \
	else \
		rm -f "ioping.plugin.tmp"; \
		false; \
	fi
Making all in freebsd.plugin
make[3]: Nothing to be done for `all'.
Making all in freeipmi.plugin
make[3]: Nothing to be done for `all'.
Making all in idlejitter.plugin
make[3]: Nothing to be done for `all'.
Making all in macos.plugin
make[3]: Nothing to be done for `all'.
Making all in nfacct.plugin
make[3]: Nothing to be done for `all'.
Making all in xenstat.plugin
make[3]: Nothing to be done for `all'.
Making all in perf.plugin
make[3]: Nothing to be done for `all'.
Making all in node.d.plugin
if sed \
		-e 's#[@]localstatedir_POST@#/usr/local/netdata/var#g' \
		-e 's#[@]sbindir_POST@#/usr/local/netdata/usr/sbin#g' \
		-e 's#[@]pluginsdir_POST@#/usr/local/netdata/usr/libexec/netdata/plugins.d#g' \
		-e 's#[@]configdir_POST@#/usr/local/netdata/etc/netdata#g' \
		-e 's#[@]libconfigdir_POST@#/usr/local/netdata/usr/lib/netdata/conf.d#g' \
		-e 's#[@]cachedir_POST@#/usr/local/netdata/var/cache/netdata#g' \
		-e 's#[@]registrydir_POST@#/usr/local/netdata/var/lib/netdata/registry#g' \
		-e 's#[@]varlibdir_POST@#/usr/local/netdata/var/lib/netdata#g' \
		node.d.plugin.in > node.d.plugin.tmp; then \
		mv "node.d.plugin.tmp" "node.d.plugin"; \
	else \
		rm -f "node.d.plugin.tmp"; \
		false; \
	fi
Making all in proc.plugin
make[3]: Nothing to be done for `all'.
Making all in python.d.plugin
if sed \
		-e 's#[@]localstatedir_POST@#/usr/local/netdata/var#g' \
		-e 's#[@]sbindir_POST@#/usr/local/netdata/usr/sbin#g' \
		-e 's#[@]pluginsdir_POST@#/usr/local/netdata/usr/libexec/netdata/plugins.d#g' \
		-e 's#[@]configdir_POST@#/usr/local/netdata/etc/netdata#g' \
		-e 's#[@]libconfigdir_POST@#/usr/local/netdata/usr/lib/netdata/conf.d#g' \
		-e 's#[@]cachedir_POST@#/usr/local/netdata/var/cache/netdata#g' \
		-e 's#[@]registrydir_POST@#/usr/local/netdata/var/lib/netdata/registry#g' \
		-e 's#[@]varlibdir_POST@#/usr/local/netdata/var/lib/netdata#g' \
		python.d.plugin.in > python.d.plugin.tmp; then \
		mv "python.d.plugin.tmp" "python.d.plugin"; \
	else \
		rm -f "python.d.plugin.tmp"; \
		false; \
	fi
Making all in slabinfo.plugin
make[3]: Nothing to be done for `all'.
Making all in statsd.plugin
make[3]: Nothing to be done for `all'.
Making all in tc.plugin
if sed \
		-e 's#[@]localstatedir_POST@#/usr/local/netdata/var#g' \
		-e 's#[@]sbindir_POST@#/usr/local/netdata/usr/sbin#g' \
		-e 's#[@]pluginsdir_POST@#/usr/local/netdata/usr/libexec/netdata/plugins.d#g' \
		-e 's#[@]configdir_POST@#/usr/local/netdata/etc/netdata#g' \
		-e 's#[@]libconfigdir_POST@#/usr/local/netdata/usr/lib/netdata/conf.d#g' \
		-e 's#[@]cachedir_POST@#/usr/local/netdata/var/cache/netdata#g' \
		-e 's#[@]registrydir_POST@#/usr/local/netdata/var/lib/netdata/registry#g' \
		-e 's#[@]varlibdir_POST@#/usr/local/netdata/var/lib/netdata#g' \
		tc-qos-helper.sh.in > tc-qos-helper.sh.tmp; then \
		mv "tc-qos-helper.sh.tmp" "tc-qos-helper.sh"; \
	else \
		rm -f "tc-qos-helper.sh.tmp"; \
		false; \
	fi
make[3]: Nothing to be done for `all-am'.
Making all in daemon
if sed \
		-e 's#[@]localstatedir_POST@#/usr/local/netdata/var#g' \
		-e 's#[@]sbindir_POST@#/usr/local/netdata/usr/sbin#g' \
		-e 's#[@]pluginsdir_POST@#/usr/local/netdata/usr/libexec/netdata/plugins.d#g' \
		-e 's#[@]configdir_POST@#/usr/local/netdata/etc/netdata#g' \
		-e 's#[@]libconfigdir_POST@#/usr/local/netdata/usr/lib/netdata/conf.d#g' \
		-e 's#[@]cachedir_POST@#/usr/local/netdata/var/cache/netdata#g' \
		-e 's#[@]registrydir_POST@#/usr/local/netdata/var/lib/netdata/registry#g' \
		-e 's#[@]varlibdir_POST@#/usr/local/netdata/var/lib/netdata#g' \
		anonymous-statistics.sh.in > anonymous-statistics.sh.tmp; then \
		mv "anonymous-statistics.sh.tmp" "anonymous-statistics.sh"; \
	else \
		rm -f "anonymous-statistics.sh.tmp"; \
		false; \
	fi
Making all in database
Making all in engine
make[3]: Nothing to be done for `all'.
make[3]: Nothing to be done for `all-am'.
Making all in health
Making all in notifications
if sed \
		-e 's#[@]localstatedir_POST@#/usr/local/netdata/var#g' \
		-e 's#[@]sbindir_POST@#/usr/local/netdata/usr/sbin#g' \
		-e 's#[@]pluginsdir_POST@#/usr/local/netdata/usr/libexec/netdata/plugins.d#g' \
		-e 's#[@]configdir_POST@#/usr/local/netdata/etc/netdata#g' \
		-e 's#[@]libconfigdir_POST@#/usr/local/netdata/usr/lib/netdata/conf.d#g' \
		-e 's#[@]cachedir_POST@#/usr/local/netdata/var/cache/netdata#g' \
		-e 's#[@]registrydir_POST@#/usr/local/netdata/var/lib/netdata/registry#g' \
		-e 's#[@]varlibdir_POST@#/usr/local/netdata/var/lib/netdata#g' \
		alarm-notify.sh.in > alarm-notify.sh.tmp; then \
		mv "alarm-notify.sh.tmp" "alarm-notify.sh"; \
	else \
		rm -f "alarm-notify.sh.tmp"; \
		false; \
	fi
make[3]: Nothing to be done for `all-am'.
Making all in libnetdata
Making all in adaptive_resortable_list
make[3]: Nothing to be done for `all'.
Making all in avl
make[3]: Nothing to be done for `all'.
Making all in buffer
make[3]: Nothing to be done for `all'.
Making all in clocks
make[3]: Nothing to be done for `all'.
Making all in config
make[3]: Nothing to be done for `all'.
Making all in dictionary
make[3]: Nothing to be done for `all'.
Making all in eval
make[3]: Nothing to be done for `all'.
Making all in json
make[3]: Nothing to be done for `all'.
Making all in health
make[3]: Nothing to be done for `all'.
Making all in locks
make[3]: Nothing to be done for `all'.
Making all in log
make[3]: Nothing to be done for `all'.
Making all in popen
make[3]: Nothing to be done for `all'.
Making all in procfile
make[3]: Nothing to be done for `all'.
Making all in simple_pattern
make[3]: Nothing to be done for `all'.
Making all in socket
make[3]: Nothing to be done for `all'.
Making all in statistical
make[3]: Nothing to be done for `all'.
Making all in storage_number
make[3]: Nothing to be done for `all'.
Making all in threads
make[3]: Nothing to be done for `all'.
Making all in url
make[3]: Nothing to be done for `all'.
make[3]: Nothing to be done for `all-am'.
Making all in registry
make[2]: Nothing to be done for `all'.
Making all in streaming
make[2]: Nothing to be done for `all'.
Making all in web
Making all in api
Making all in badges
make[4]: Nothing to be done for `all'.
Making all in queries
Making all in average
make[5]: Nothing to be done for `all'.
Making all in des
make[5]: Nothing to be done for `all'.
Making all in incremental_sum
make[5]: Nothing to be done for `all'.
Making all in max
make[5]: Nothing to be done for `all'.
Making all in min
make[5]: Nothing to be done for `all'.
Making all in sum
make[5]: Nothing to be done for `all'.
Making all in median
make[5]: Nothing to be done for `all'.
Making all in ses
make[5]: Nothing to be done for `all'.
Making all in stddev
make[5]: Nothing to be done for `all'.
make[5]: Nothing to be done for `all-am'.
Making all in exporters
Making all in shell
make[5]: Nothing to be done for `all'.
Making all in prometheus
make[5]: Nothing to be done for `all'.
make[5]: Nothing to be done for `all-am'.
Making all in formatters
Making all in csv
make[5]: Nothing to be done for `all'.
Making all in json
make[5]: Nothing to be done for `all'.
Making all in ssv
make[5]: Nothing to be done for `all'.
Making all in value
make[5]: Nothing to be done for `all'.
make[5]: Nothing to be done for `all-am'.
Making all in health
make[4]: Nothing to be done for `all'.
make[4]: Nothing to be done for `all-am'.
Making all in gui
if test -f dashboard.js; then rm -f dashboard.js; fi
cat src/dashboard.js/prologue.js.inc src/dashboard.js/utils.js src/dashboard.js/server-detection.js src/dashboard.js/dependencies.js src/dashboard.js/error-handling.js src/dashboard.js/compatibility.js src/dashboard.js/xss.js src/dashboard.js/colors.js src/dashboard.js/units-conversion.js src/dashboard.js/options.js src/dashboard.js/localstorage.js src/dashboard.js/timeout.js src/dashboard.js/themes.js src/dashboard.js/charting/dygraph.js src/dashboard.js/charting/sparkline.js src/dashboard.js/charting/google-charts.js src/dashboard.js/charting/gauge.js src/dashboard.js/charting/easy-pie-chart.js src/dashboard.js/charting/d3pie.js src/dashboard.js/charting/d3.js src/dashboard.js/charting/peity.js src/dashboard.js/charting/textonly.js src/dashboard.js/charting.js src/dashboard.js/chart-registry.js src/dashboard.js/common.js src/dashboard.js/main.js src/dashboard.js/alarms.js src/dashboard.js/registry.js src/dashboard.js/boot.js src/dashboard.js/epilogue.js.inc  > dashboard.js.tmp && mv dashboard.js.tmp dashboard.js
if test -d "../../.git"; then \
		git --git-dir="../../.git" log -n 1 --format=%H; \
	fi > version.txt.tmp
test -s version.txt.tmp || echo 0 > version.txt.tmp
mv version.txt.tmp version.txt
Making all in server
Making all in static
make[5]: Nothing to be done for `all-am'.
make[4]: Nothing to be done for `all-am'.
make[3]: Nothing to be done for `all-am'.
  CC       collectors/cups.plugin/cups_plugin.o
  CC       libnetdata/adaptive_resortable_list/adaptive_resortable_list.o
  CC       libnetdata/config/appconfig.o
  CC       libnetdata/avl/avl.o
  CC       libnetdata/buffer/buffer.o
  CC       libnetdata/clocks/clocks.o
  CC       libnetdata/dictionary/dictionary.o
  CC       libnetdata/eval/eval.o
  CC       libnetdata/libnetdata.o
  CC       libnetdata/locks/locks.o
  CC       libnetdata/log/log.o
  CC       libnetdata/popen/popen.o
  CC       libnetdata/procfile/procfile.o
  CC       libnetdata/os.o
  CC       libnetdata/simple_pattern/simple_pattern.o
  CC       libnetdata/socket/socket.o
  CC       libnetdata/socket/security.o
  CC       libnetdata/statistical/statistical.o
  CC       libnetdata/storage_number/storage_number.o
  CC       libnetdata/threads/threads.o
  CC       libnetdata/url/url.o
  CC       libnetdata/json/json.o
libnetdata/json/json.c:318:17: warning: address of array 'e->name' will always
      evaluate to 'true' [-Wpointer-bool-conversion]
        if (!e->name || !e->fullname || strlen(e->name) > JSON_NAME_LE...
            ~~~~^~~~
libnetdata/json/json.c:318:29: warning: address of array 'e->fullname' will
      always evaluate to 'true' [-Wpointer-bool-conversion]
        if (!e->name || !e->fullname || strlen(e->name) > JSON_NAME_LE...
                        ~~~~^~~~~~~~
2 warnings generated.
  CC       libnetdata/json/jsmn.o
  CC       libnetdata/health/health.o
libnetdata/health/health.c:115:19: warning: address of array 'e->name' will
      always evaluate to 'true' [-Wpointer-bool-conversion]
            if(e->name && strcmp(e->name,"")) {
               ~~~^~~~ ~~
1 warning generated.
  CCLD     cups.plugin
ld: warning: directory not found for option '-L/usr/local/opt/openssl/lib'
  CC       daemon/common.o
  CC       daemon/daemon.o
daemon/daemon.c:63:40: warning: passing 'gid_t *' (aka 'unsigned int *') to
      parameter of type 'int *' converts between pointers to integer types with
      different sign [-Wpointer-sign]
        if(getgrouplist(username, gid, supplementary_groups, &ngroups) == -1) {
                                       ^~~~~~~~~~~~~~~~~~~~
/Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk/usr/include/unistd.h:653:43: note: 
      passing argument to parameter here
int      getgrouplist(const char *, int, int *, int *);
                                              ^
1 warning generated.
  CC       daemon/global_statistics.o
daemon/global_statistics.c:159:68: warning: passing 'volatile uint64_t *' (aka
      'volatile unsigned long long *') to parameter of type 'uint64_t *' (aka
      'unsigned long long *') discards qualifiers
      [-Wincompatible-pointer-types-discards-qualifiers]
  ...&gs->web_usec_max, &n, 1, __ATOMIC_SEQ_CST,
     ^~~~~~~~~~~~~~~~~
1 warning generated.
  CC       daemon/main.o
  CC       daemon/signals.o
  CC       daemon/unit_test.o
  CC       web/api/badges/web_buffer_svg.o
  CC       web/api/exporters/allmetrics.o
  CC       web/api/exporters/shell/allmetrics_shell.o
  CC       web/api/queries/average/average.o
  CC       web/api/queries/des/des.o
  CC       web/api/queries/incremental_sum/incremental_sum.o
  CC       web/api/queries/max/max.o
  CC       web/api/queries/median/median.o
  CC       web/api/queries/min/min.o
  CC       web/api/queries/query.o
  CC       web/api/queries/rrdr.o
  CC       web/api/queries/ses/ses.o
  CC       web/api/queries/stddev/stddev.o
  CC       web/api/queries/sum/sum.o
  CC       web/api/formatters/rrd2json.o
  CC       web/api/formatters/csv/csv.o
  CC       web/api/formatters/json/json.o
  CC       web/api/formatters/ssv/ssv.o
  CC       web/api/formatters/value/value.o
  CC       web/api/formatters/json_wrapper.o
  CC       web/api/formatters/charts2json.o
  CC       web/api/formatters/rrdset2json.o
  CC       web/api/health/health_cmdapi.o
  CC       web/api/web_api_v1.o
  CC       backends/backends.o
  CC       backends/graphite/graphite.o
  CC       backends/json/json.o
  CC       backends/opentsdb/opentsdb.o
  CC       backends/prometheus/backend_prometheus.o
  CC       collectors/checks.plugin/plugin_checks.o
  CC       health/health.o
  CC       health/health_config.o
  CC       health/health_json.o
  CC       health/health_log.o
health/health_log.c:116:31: warning: format specifies type 'unsigned long' but
      the argument has type 'uint64_t' (aka 'unsigned long long') [-Wformat]
                            , (uint64_t)ae->last_repeat
                              ^~~~~~~~~~~~~~~~~~~~~~~~~
./config.h:359:41: note: expanded from macro 'unlikely'
#define unlikely(x) __builtin_expect(!!(x), 0)
                                        ^
1 warning generated.
  CC       collectors/idlejitter.plugin/plugin_idlejitter.o
  CC       collectors/plugins.d/plugins_d.o
  CC       registry/registry.o
  CC       registry/registry_db.o
  CC       registry/registry_init.o
  CC       registry/registry_internals.o
  CC       registry/registry_log.o
  CC       registry/registry_machine.o
  CC       registry/registry_person.o
  CC       registry/registry_url.o
  CC       database/rrdcalc.o
  CC       database/rrdcalctemplate.o
  CC       database/rrddim.o
  CC       database/rrddimvar.o
  CC       database/rrdfamily.o
  CC       database/rrdhost.o
  CC       database/rrd.o
  CC       database/rrdset.o
  CC       database/rrdsetvar.o
  CC       database/rrdvar.o
  CC       streaming/rrdpush.o
  CC       collectors/statsd.plugin/statsd.o
  CC       web/server/web_client.o
  CC       web/server/web_server.o
  CC       web/server/web_client_cache.o
  CC       web/server/static/static-threaded.o
  CC       collectors/macos.plugin/plugin_macos.o
  CC       collectors/macos.plugin/macos_sysctl.o
  CC       collectors/macos.plugin/macos_mach_smi.o
  CC       collectors/macos.plugin/macos_fw.o
collectors/macos.plugin/macos_fw.c:118:35: warning: address of array
      'diskstat.name' will always evaluate to 'true' [-Wpointer-bool-conversion]
            if(unlikely(!diskstat.name || !*diskstat.name)) {
                        ~~~~~~~~~~^~~~
./config.h:359:41: note: expanded from macro 'unlikely'
#define unlikely(x) __builtin_expect(!!(x), 0)
                                        ^
1 warning generated.
  GEN      netdata
ld: warning: directory not found for option '-L/usr/local/opt/openssl/lib'
iMac:netdata jean-nicolasreyt$

--- a/netdata-installer.sh
+++ b/netdata-installer.sh
@@ -80,6 +80,7 @@ umask 002
 renice 19 $$ >/dev/null 2>/dev/null
 
 # you can set CFLAGS before running installer
+LDFLAGS="${LDFLAGS}"
 CFLAGS="${CFLAGS--O2}"
 [ "z${CFLAGS}" = "z-O3" ] && CFLAGS="-O2"
 
@@ -88,6 +89,7 @@ CFLAGS="${CFLAGS--O2}"
 printf "\\n# " >>netdata-installer.log
 date >>netdata-installer.log
 printf 'CFLAGS="%s" ' "${CFLAGS}" >>netdata-installer.log
+printf 'LDFLAGS="%s" ' "${LDFLAGS}" >>netdata-installer.log
 printf "%q " "${PROGRAM}" "${@}" >>netdata-installer.log
 printf "\\n" >>netdata-installer.log
 
@@ -182,6 +184,14 @@ Netdata will by default be compiled with gcc optimization -O2
 If you need to pass different CFLAGS, use something like this:
 
   CFLAGS="<gcc options>" ${PROGRAM} [options]
+
+If you also need to provide different LDFLAGS, use something like this:
+
+  LDFLAGS="<extra ldflag options>" ${PROGRAM} [options]
+
+or use the following if both LDFLAGS and CFLAGS need to be overriden:
+
+  CFLAGS="<gcc options>" LDFLAGS="<extra ld options>" ${PROGRAM} [options]
 
 For the installer to complete successfully, you will need these packages installed:
 
@@ -405,7 +415,7 @@ run ./configure \
 	--with-math \
 	--with-user=netdata \
 	${NETDATA_CONFIGURE_OPTIONS} \
-	CFLAGS="${CFLAGS}" || exit 1
+	CFLAGS="${CFLAGS}" LDFLAGS="${LDFLAGS}" || exit 1
 
 # remove the build_error hook
 trap - EXIT
@@ -1115,6 +1125,7 @@ cat <<EOF > "${NETDATA_USER_CONFIG_DIR}/.environment"
 # Created by installer
 PATH="${PATH}"
 CFLAGS="${CFLAGS}"
+LDFLAGS="${LDFLAGS}"
 NETDATA_PREFIX="${NETDATA_PREFIX}"
 NETDATA_CONFIGURE_OPTIONS="${NETDATA_CONFIGURE_OPTIONS}"
 NETDATA_ADDED_TO_GROUPS="${NETDATA_ADDED_TO_GROUPS}"

--- a/packaging/installer/README.md
+++ b/packaging/installer/README.md
@@ -428,7 +428,7 @@ sudo ./netdata-installer.sh --install /usr/local
 The installer will also install a startup plist to start Netdata when your Mac boots.
 
 **Note:** Should you wish to install Netdata with TLS support:
-1. Install open SSL using brew, by executing `brew install openssl`
+1. Install OpenSSL via brew by executing `brew install openssl`
 2. Run the installer with the extra CFLAGS and LDFLAGS, since open SSL installation is not automatically symlinked to /usr/local
 ```sh
 # install Netdata in /usr/local/netdata

--- a/packaging/installer/README.md
+++ b/packaging/installer/README.md
@@ -430,6 +430,7 @@ The installer will also install a startup plist to start Netdata when your Mac b
 **Note:** Should you wish to install Netdata with TLS support:
 1. Install OpenSSL via brew by executing `brew install openssl`
 2. Run the installer with the extra CFLAGS and LDFLAGS, since your OpenSSL installation is not automatically symlinked to `/usr/local`
+
 ```sh
 # install Netdata in /usr/local/netdata
 cd netdata

--- a/packaging/installer/README.md
+++ b/packaging/installer/README.md
@@ -427,6 +427,16 @@ sudo ./netdata-installer.sh --install /usr/local
 
 The installer will also install a startup plist to start Netdata when your Mac boots.
 
+**Note:** Should you wish to install netdata with open SSL support:
+1. Install open SSL using brew, by executing `brew install openssl`
+2. Run the installer with the extra CFLAGS and LDFLAGS, since open SSL installation is not automatically symlinked to /usr/local
+```sh
+# install Netdata in /usr/local/netdata
+cd netdata
+CFLAGS="-I$(brew --prefix)/opt/openssl/include" LDFLAGS="${LDFLAGS} -L$(brew --prefix)/opt/openssl/lib" sudo -E ./netdata-installer.sh --install /usr/local
+```
+
+
 ##### Alpine 3.x
 
 Execute these commands to install Netdata in Alpine Linux 3.x:

--- a/packaging/installer/README.md
+++ b/packaging/installer/README.md
@@ -429,7 +429,7 @@ The installer will also install a startup plist to start Netdata when your Mac b
 
 **Note:** Should you wish to install Netdata with TLS support:
 1. Install OpenSSL via brew by executing `brew install openssl`
-2. Run the installer with the extra CFLAGS and LDFLAGS, since open SSL installation is not automatically symlinked to /usr/local
+2. Run the installer with the extra CFLAGS and LDFLAGS, since your OpenSSL installation is not automatically symlinked to `/usr/local`
 ```sh
 # install Netdata in /usr/local/netdata
 cd netdata

--- a/packaging/installer/README.md
+++ b/packaging/installer/README.md
@@ -427,7 +427,7 @@ sudo ./netdata-installer.sh --install /usr/local
 
 The installer will also install a startup plist to start Netdata when your Mac boots.
 
-**Note:** Should you wish to install netdata with open SSL support:
+**Note:** Should you wish to install Netdata with TLS support:
 1. Install open SSL using brew, by executing `brew install openssl`
 2. Run the installer with the extra CFLAGS and LDFLAGS, since open SSL installation is not automatically symlinked to /usr/local
 ```sh

--- a/packaging/installer/netdata-updater.sh
+++ b/packaging/installer/netdata-updater.sh
@@ -6,6 +6,7 @@
 # Variables needed by script:
 #  - PATH
 #  - CFLAGS
+#  - LDFLAGS
 #  - NETDATA_CONFIGURE_OPTIONS
 #  - REINSTALL_COMMAND
 #  - NETDATA_TARBALL_URL

